client/components/forms/form-section-heading/index.jsx

```diff
@@ -1,7 +1,16 @@
 import classnames from 'classnames';
+import { ReactChild, ReactElement } from 'react';
```

Comment:
Technically we don't use those in this component.

Could you instead use the `@param {import('react').ReactChild}` syntax instead and avoid importing them unnecessarily here?

Reply:
Great point, thanks @tyxla! I've created [a follow-up PR to handle this](https://github.com/Automattic/wp-calypso/pull/60134), please review.